src/sentry/api/serializers/models/group.py

```diff
@@ -179,6 +182,24 @@ def __init__(
         self.collapse = collapse
         self.expand = expand
 
+    def _serialize_assigness(
+        self, actor_dict: Mapping[int, ActorTuple]
+    ) -> Mapping[int, Union[Team, Any]]:
+        actors_by_type: MutableMapping[Any, List[ActorTuple]] = defaultdict(list)
+        for actor in actor_dict.values():
+            actors_by_type[actor.type].append(actor)
+
+        resolved_actors = {}
+        for t, actors in actors_by_type.items():
+            serializable = ActorTuple.resolve_many(actors)
+            if t == Team:
+                resolved_actors[t] = {actor.id: actor for actor in serializable}
+            if t == User:
+                serialized = user_service.serialize_users([u.id for u in serializable])
```

Comment:
So I now realize why I was confused before.  I was focused on the fact this is shoved in `assigned_to` but was missing the fact that this ended up in the `serialize` call with an explicit Serializer.  So I made two changes trying to fix the same issue, but this change isn't necessary actually.  Here's my suggestion:

Try removing this conditional.  `resolved_actors[t] = {actor.id: actor for actor in serializable}` is sufficient for both cases (no need for the serialize_users call, I was wrong).  Then try removing your conditional to the serializer.  I think that should work due to the `ActorSerializer` changes.